### PR TITLE
Update German translations

### PR DESCRIPTION
**Pull request by diwako**
**When merged this pull request will:**
- Update missing German translations

As usual, always welcome for other Germans to look through as I was never in service and might get some jargon wrong.

### CODE REVIEW

File: addons/ai/stringtable.xml [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <Package name="AI">
         <Key ID="STR_ZEN_AI_Paradrop">
             <English>Paradrop</English>
+            <German>Paradrop</German>
```
Comment (bux) [inline addons/ai/stringtable.xml]: Ist das jetzt deutsch? :)

File: addons/attributes/stringtable.xml [hunk 2]
```diff
@@ -37,60 +37,79 @@
...
             <English>Commanding</English>
+            <German>Kommandierend</German>
         </Key>
         <Key ID="STR_ZEN_Attributes_Courage">
             <English>Courage</English>
+            <German>Mut</German>
         </Key>
         <Key ID="STR_ZEN_Attributes_SpotDistance">
             <English>Spot Distance</English>
+            <German>Spottdistanz</German>
```
Comment (bux) [inline addons/attributes/stringtable.xml]: "Spot" steht hier eher für "Aufklären"

File: addons/editor/stringtable.xml [hunk 3]
```diff
@@ -73,15 +73,19 @@
         </Key>
         <Key ID="STR_ZEN_Editor_EjectPassengers">
             <English>Eject Passengers</English>
+            <German>Passagiere rauswerfen</German>
         </Key>
         <Key ID="STR_ZEN_Editor_EjectPassengers_Description">
             <English>Ejects the passengers of all selected vehicles. If the vehicle is currently flying, the units will be given parachutes.\nUnlike vanilla eject, units will disembark one after another (not immediately, with a short delay in-between).</English>
+            <German>Wirft die Fahrgäste aller ausgewählten Fahrzeuge raus. Wenn das Fahrzeug gerade fliegt, erhalten die Einheiten Fallschirme. Im Gegensatz zum Vanilla auswurf werden die Einheiten nacheinander aussteigen (nicht alle sofort, sondern mit einer kurzen Verzögerung dazwischen).</German>
```
Comment (bux) [inline addons/editor/stringtable.xml]: Vanilla auswurf?

File: addons/modules/stringtable.xml [hunk 4]
```diff
@@ -1197,81 +1201,109 @@
...
             <English>Sets the rain strength, requires overcast to be greater than 70%.</English>
+            <German>Legt die Regenstärke fest, erfordert, dass die Bewölkung größer als 70% ist.</German>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleWeather_Lightning">
             <English>Lightning</English>
+            <German>Blitzschlag</German>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleWeather_Lightning_Tooltip">
             <English>Sets the frequency of lightning strikes, requires overcast to be near 100%.</English>
+            <German>Sets the frequency of lightning strikes, requires overcast to be near 100%.</German>
```
Comment (bux) [inline addons/modules/stringtable.xml]: TODO

File: addons/modules/stringtable.xml [hunk 5]
```diff
@@ -1197,81 +1201,109 @@
...
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleWeather_FogDecay">
             <English>Fog Decay</English>
+            <German>Nebelzerfall</German>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleWeather_FogDecay_Tooltip">
             <English>Sets how much the fog density decays with altitude. Larger values result in a sharper fog ceiling (more defined fog start).\nSmaller values lead to a more smooth transition from no fog to full fog, and as a result, the fog will cross the base altitude line more.\nWhen this is a positive value, the fog will be generated below the base altitude (valley fog); if negative, above the base altitude (mountain fog).</English>
+            <German>Legt fest, wie stark die Nebeldichte mit der Höhe abnimmt. Größere Werte führen zu einer schärferen Nebeldecke (definierterer Nebelbeginn). \nKleinere Werte führen zu einem sanfteren Übergang von keinem Nebel zu vollem Nebel, so dass der Nebel die Basishöhenlinie mehr überschreitet. \nWenn dies ein positiver Wert ist, wird der Nebel unterhalb der Basishöhe (Talnebel) erzeugt, wenn negativ, oberhalb der Basishöhe (Bergnebel).
+
+Übersetzt mit www.DeepL.com/Translator</German>
```
Comment (bux) [inline addons/modules/stringtable.xml]: deepl

File: addons/editor/stringtable.xml (hunk 3 above)
Comment (diwako) [reply addons/editor/stringtable.xml]: Not sure aobut that one, i would translate "eject" to "Auswurf" or "Rauswruf".

File: addons/ai/stringtable.xml (hunk 1 above)
Comment (diwako) [reply addons/ai/stringtable.xml]: how about `Fallschirmsprung` ?

File: addons/editor/stringtable.xml (hunk 3 above)
Comment (bux) [reply addons/editor/stringtable.xml]: maybe `Im Gegensatz zum "Vanilla" sitzen Einheiten nacheinander ab`

**Comment by mharis001** [comment]
Thank you for translating, and thanks @bux for reviewing.

**Pull request merged**